protocol/contracts/zaps/GenericZap.sol

```diff
@@ -192,7 +192,7 @@ contract GenericZap is ZapBase {
     address swapTarget,
```

Comment:
At the top of this contract -- In order for this to work in testnet, would have to change:
```
  ICurveFactory private immutable curveFactory = ICurveFactory(0xB9fC157394Af804a3578134A6585C0dc9cc990d4);
  IBalancerVault private immutable balancerVault = IBalancerVault(0xBA12222222228d8Ba445958a75a0704d566BF2C8);
```
To be set in the constructor too...know what the goerli addresses are (if any)?

Reply:
`balancerVault` can be found here
https://goerli.etherscan.io/address/0xBA12222222228d8Ba445958a75a0704d566BF2C8

Reply:
Will 0x on goerli have issues routing to the right pools if they're self-created?